### PR TITLE
feat: add privacy policy zh

### PR DESCRIPTION
**Pull request by liuxiran**
1. Add privacy policy menu at the footer

image

2. Add privacy policy page refer to our privacy policy doc, only for Chinese now, since the English version is still under proofreading.

### CODE REVIEW

File: _posts/page/zh-CN/privacy_policy.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+* 我们在控制权等方面发生重大变化。如并购重组等引起的信息控制者变更等。
+* 个人信息共享、转让或公开披露的主要对象发生变化。
+* 您参与个人信息处理方面的权利及其行使方式发生重大变化。
+* 我们负责处理个人信息安全的责任部门、联络方式及投诉渠道发生变化。
+* 个人信息安全影响评估报告表明存在高风险。
+* 我们还会将本隐私权政策的旧版本在我们的官方网站存档，供您查阅。
+
+## 十二、如何联系我们
+
+如您对本政策或其他相关事宜有疑问，请根据我们提供的指引，填写相关资料，将您的问题发送至 support@api7.ai 或通过 [公司的官方网站](https://www.apiseven.com/) 联系我们。我们将尽快审核所涉问题，并在验证您的用户身份后的三十天内予以回复。
```
Comment (juzhiyuan) [inline _posts/page/zh-CN/privacy_policy.md:162]:
```suggestion
如您对本政策或其他相关事宜有疑问，请根据我们提供的指引，填写相关资料，将您的问题发送至 [mailto:support@api7.ai](support@api7.ai) 或通过 [公司的官方网站](https://www.apiseven.com/) 联系我们。我们将尽快审核所涉问题，并在验证您的用户身份后的三十天内予以回复。
```

Comment (juzhiyuan) [reply _posts/page/zh-CN/privacy_policy.md:162]: Not sure if this is working

**Comment by juzhiyuan** [comment]
image

This position is a little wired 🤣

File: _posts/page/zh-CN/privacy_policy.md (hunk 1 above)
Comment (liuxiran) [reply _posts/page/zh-CN/privacy_policy.md:162]: thanks @juzhiyuan , actually the current format can also arouse the email .

image


refer to https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses, I should add `<>`

and refer to https://github.com/github/markup/issues/1030#issuecomment-288540993, we can also add a subject to the email.

Let me update it.

File: _posts/page/zh-CN/privacy_policy.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+5. 您自行向社会公众公开的个人信息。
+6. 从合法公开披露的信息中收集个人信息的，如合法的新闻报道、政府信息公开等渠道。
+7. 用于维护产品及相关服务的安全稳定运行所必需的，例如发现、处置产品及相关服务的故障。
+8. 根据法律规定，共享、转让经去标识化处理的个人信息，且确保数据接收方无法复原并重新识别个人信息主体的，不属于个人信息的对外共享、转让及公开披露行为，对此类数据的保存及处理将无需另行向您通知并征得您的同意。
+
+特别提示您注意，如信息无法单独或结合其他信息识别到您的个人身份，其不属于法律意义上您的个人信息；当您的信息可以单独或结合其他信息识别到您的个人身份时或我们将无法与任何特定个人信息建立联系的数据与其他您的个人信息结合使用时，这些信息在结合使用期间，将作为您的个人信息按照本隐私政策处理与保护。
+
+## 九、处理未成年人的个人信息
+
+如果没有父母或监护人的同意，未成年人不得创建自己的用户账号。如您为未成年人的，建议您请您的父母或监护人仔细阅读本隐私政策，并在征得您的父母或监护人同意的前提下使用我们的服务或向我们提供信息。对于经父母或监护人同意使用我们的产品或服务而收集未成年人个人信息的情况，我们只会在法律法规允许、父母或监护人明确同意或者保护未成年人所必要的情况下使用、共享、转让或披露此信息。
```
Comment (LiteSun) [inline _posts/page/zh-CN/privacy_policy.md:140]:
```suggestion
如果没有父母或监护人的同意，未成年人不得创建自己的用户账号。如您为未成年人，建议您请您的父母或监护人仔细阅读本隐私政策，并在征得您的父母或监护人同意的前提下使用我们的服务或向我们提供信息。对于经父母或监护人同意使用我们的产品或服务而收集未成年人个人信息的情况，我们只会在法律法规允许、父母或监护人明确同意或者保护未成年人所必要的情况下使用、共享、转让或披露此信息。
```

File: _posts/page/zh-CN/privacy_policy.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+* 内容和通讯：您在表格上的任何“自由文本”框中输入的任何数据，聊天表格中的评论以及在社交媒体和其他平台（例如GitHub和Twitter）上通过电话或IM服务（例如，微信，Slack等）。
+
+**信息自动收集**。如下面的“我们对Cookie和其他技术的使用”和“产品遥测数据”部分中进一步所述，当您使用我们的网站或产品时，我们和我们的第三方合作伙伴会自动收集某些类型的设备和使用信息。我们收集的信息包括，例如：
+
+* 地理位置数据：当您访问我们的网站并使用我们的产品时，我们会根据您的设备设置收集地理位置数据。例如，我们可能会使用IP地址得出您的大致位置。
+* 标识符和设备信息：当您访问我们的网站时，您的设备的操作系统，设备标识符，客户ID和其他设备信息，用户代理字符串，Internet协议（IP）地址，访问时间，浏览器类型以及详细记录您与之交互的日志数据我们的网站（例如，点击次数，浏览的页面，搜索到的信息）以及您访问我们网站之前和之后访问的网站（即引荐来源标头）都会自动记录下来。当您提交支持票证时，我们将从您的浏览器的用户代理字符串中接收数据，其中包括设备信息，浏览器信息，操作系统信息，城市，省份，国家和IP地址。
+
+**推论**。我们从收集到的数据中推断出有关您和您的公司的新信息，包括使用自动方式生成有关您可能的偏好，服务和产品需求或其他特征的信息。例如，我们根据您的IP地址推断您的城市，省和国家/地区位置。
+
+**通过社交媒体和其他平台收集的信息**。当您通过各种第三方平台与我们互动时，我们会收到有关您的信息，例如，通过加入我们的Slack社区或Google小组，在Facebook上喜欢我们，在LinkedIn，GitHub或Meetup上建立联系，在Twitter上关注我们或在Facebook，Twitter或LinkedIn 上共享我们网站上的内容。我们与这些第三方平台交换的数据可能取决于您在这些平台上的隐私设置。在参与之前，您应该查看并考虑调整您在第三方网站和服务上的隐私设置。不要通过这些平台向我们提供任何敏感的个人数据。我们不对这些第三方平台的数据保护和使用实践负责。请查看他们的隐私政策，以了解他们如何使用您的信息。
```
Comment (LiteSun) [inline _posts/page/zh-CN/privacy_policy.md:32]:
```suggestion
**通过社交媒体和其他平台收集的信息**。当您通过各种第三方平台与我们互动时，我们会收到有关您的信息，例如，通过加入我们的Slack社区或Google小组，在Facebook上喜欢我们，在LinkedIn，GitHub或Meetup上建立联系，在Twitter上关注我们或在Facebook，Twitter或LinkedIn上共享我们网站上的内容。我们与这些第三方平台交换的数据可能取决于您在这些平台上的隐私设置。在参与之前，您应该查看并考虑调整您在第三方网站和服务上的隐私设置。不要通过这些平台向我们提供任何敏感的个人数据。我们不对这些第三方平台的数据保护和使用实践负责。请查看他们的隐私政策，以了解他们如何使用您的信息。
```

**Comment by liuxiran** [comment]
> This position is a little wired 🤣

ok, I briefly compared a few sites, at last I choose Kong style 
image

maybe we have two line about our copyright, It seems a little wired

Let me offer more options

File: _posts/page/zh-CN/privacy_policy.md (hunk 3 above)
Comment (liuxiran) [reply _posts/page/zh-CN/privacy_policy.md]: thanks, fixed together with other typos

File: _posts/page/zh-CN/privacy_policy.md (hunk 1 above)
Comment (liuxiran) [reply _posts/page/zh-CN/privacy_policy.md]: fixed thanks

**Comment by juzhiyuan** [comment]
会显得比较突兀 🤣

**Comment by liuxiran** [comment]
updated the location and it can only be found at footer

image

**Review by juzhiyuan — APPROVED** [review]
内容上看着没啥问题，隐私条款 链接放在 公司分类 下是否会更合适？（不确定）

**Comment by liuxiran** [comment]
> 内容上看着没啥问题，隐私条款 链接放在 公司分类 下是否会更合适？（不确定）

I prefer keeping it in the resources group, since the privacy policy is a part of the website resources IMO. Just wait for others' option. ^^

cc @moonming @membphis @LiteSun

**Comment by LiteSun** [comment]
After referring to the location of privacy agreements on several sites, I think privacy agreements are more like a separate category.

image
image
image

**Comment by liuxiran** [comment]
> After referring to the location of privacy agreements on several sites, I think privacy agreements are more like a separate category.

The privacy policy is a part of **Legal terms**,  and it is for users to use when browsing the website, so it should be part of the resources of the website itself. If we have more legal provisions in the future, we can consider a single category [Legal terms], just like Splitbee

image


or put it together with copyright is also a good choice just like:

* Github
image

* Google search
image

* InfoQ
image

* Kong
image

* Solo
image

Here is a demo: 
image


What do you think?
@juzhiyuan @LiteSun @moonming @membphis @iamayushdas @Yiyiyimu

**Comment by iamayushdas** [comment]
I liked the demo @liuxiran

**Comment by juzhiyuan** [comment]
image

@liuxiran how about making those contents in one line?

**Comment by liuxiran** [comment]
> image
> 
> @liuxiran how about making those contents in one line?

good idea!  just see the updated demo @juzhiyuan 

image

**Comment by liuxiran** [comment]
updated:

`zh-CN`
image

`en-US`
image

@juzhiyuan @LiteSun @iamayushdas please take a look again when you have time , thanks a lot~

**Comment by juzhiyuan** [comment]
@liuxiran We don't have Privacy in English, right?

**Comment by iamayushdas** [comment]
Yes we don't have privacy policy in english

**Comment by liuxiran** [comment]
> @liuxiran We don't have Privacy in English, right?

We have , but the English version is still in the process of confirmation, it will be added later when it is ready @juzhiyuan

**Pull request merged**